Test: randoop.crystal.RandoopTest4.test153
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest4.test151]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest5.test337
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest5.test331]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest5.test481
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest3.test457]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest5.test359
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest5.test338]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest5.test230
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest5.test220]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest4.test361
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest4.test355]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest1.test166
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest0.test410]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest5.test18
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest4.test496]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest6.test76
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest4.test46]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest4.test46
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest4.test39]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest2.test100
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest1.test444]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest1.test444
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest1.test220]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest6.test109
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest5.test171]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest4.test405
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest4.test400]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest4.test216
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest4.test8]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest4.test8
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest2.test447]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest2.test447
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest2.test441]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest6.test166
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest6.test78]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest5.test379
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest5.test359]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest3.test459
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest3.test445]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest4.test78
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest4.test49]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest4.test330
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest4.test322]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest3.test102
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest2.test477]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest5.test331
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest5.test302]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest1.test222
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest1.test178]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest3.test350
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest3.test344]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest1.test132
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest1.test128]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest0.test129
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest0.test114]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest5.test128
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest5.test111]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest3.test264
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest3.test102]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest4.test145
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest4.test101]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest1.test128
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest1.test10]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest3.test409
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest3.test384]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest6.test115
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest6.test49]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest5.test473
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest4.test75]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest3.test319
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest3.test314]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest3.test314
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest3.test264]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest4.test355
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest4.test336]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest0.test292
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest0.test221]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest2.test411
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest0.test160]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest0.test93
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest0.test92]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest3.test463
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest3.test459]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest5.test138
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest5.test53]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest6.test34
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest2.test284]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest6.test55
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest5.test362]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest5.test362
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest3.test210]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest1.test55
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest0.test457]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest6.test122
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest6.test115]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest3.test472
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest3.test463]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest3.test140
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest3.test139]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest4.test80
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest0.test27]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest1.test220
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest1.test158]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest2.test2
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest1.test281]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest5.test451
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest5.test440]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest5.test440
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest5.test412]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest2.test477
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest2.test445]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest4.test336
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest4.test330]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest3.test437
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest3.test409]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest4.test322
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest4.test319]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest4.test319
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest4.test163]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest1.test10
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest0.test499]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest0.test109
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest0.test104]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest4.test421
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest3.test381]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest2.test51
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest2.test43]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest5.test313
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest4.test281]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest5.test497
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest5.test451]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest5.test338
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest5.test337]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest4.test494
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest4.test484]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest5.test382
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest5.test379]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest4.test94
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest4.test82]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest4.test82
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest4.test78]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest4.test151
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest4.test145]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest0.test104
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest0.test101]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest6.test46
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest6.test30]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest2.test308
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest2.test211]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest4.test445
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest4.test431]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest4.test101
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest4.test94]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest3.test384
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest3.test360]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest2.test201
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest2.test198]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest5.test175
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest4.test216]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest1.test51
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest0.test144]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest2.test119
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest2.test13]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest0.test92
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest0.test79]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest5.test364
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest5.test175]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest2.test441
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest2.test100]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest5.test180
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest5.test128]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest1.test158
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest1.test51]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest4.test34
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest4.test16]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest3.test487
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest3.test472]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest3.test336
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest3.test320]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest6.test30
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest5.test497]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest3.test360
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest3.test350]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest2.test198
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest2.test96]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest0.test218
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest0.test211]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest2.test445
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest2.test253]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest4.test163
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest4.test153]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest4.test49
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest4.test34]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest4.test484
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest4.test445]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest5.test208
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest5.test191]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest4.test496
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest4.test494]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest5.test412
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest5.test382]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest3.test320
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest3.test319]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest5.test488
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest5.test100]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest4.test44
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest1.test373]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest0.test400
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest0.test382]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest4.test26
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest2.test12]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest1.test281
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest1.test222]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest0.test114
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest0.test109]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest0.test221
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest0.test129]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest6.test127
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest6.test124]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest4.test16
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest3.test487]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest4.test431
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest4.test361]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest0.test79
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest0.test75]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest2.test96
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest2.test53]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest5.test220
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest5.test208]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest5.test302
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest5.test232]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest0.test75
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest0.test48]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest5.test170
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest4.test490]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest4.test490
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest4.test173]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest3.test445
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest3.test437]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest5.test80
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest5.test78]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest5.test78
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest5.test18]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest2.test253
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest2.test201]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest3.test344
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest3.test336]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest0.test382
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest0.test292]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest5.test191
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest5.test180]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest0.test101
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest0.test93]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest2.test53
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest2.test51]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest1.test178
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest1.test132]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest0.test499
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest0.test420]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest6.test147
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest6.test140]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest6.test56
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest6.test55]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest3.test210
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest1.test371]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest1.test371
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest1.test55]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest6.test72
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest6.test56]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest5.test232
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest5.test230]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest6.test140
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest6.test127]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest5.test111
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest5.test80]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest6.test124
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest6.test122]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest2.test43
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest2.test2]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest0.test420
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest0.test400]
The revealed different behavior: PASS
when executed after: []

Test: randoop.crystal.RandoopTest6.test49
Intended behavior: FAILURE
when executed after: [randoop.crystal.RandoopTest6.test46]
The revealed different behavior: PASS
when executed after: []

